unleash-yggdrasil/benches/benchmark.rs

```diff
@@ -10,18 +10,17 @@ fn is_enabled(engine: &EngineState, toggle_name: &str, context: &Context) {
 
 fn benchmark_with_no_strategy(c: &mut Criterion) {
     let mut engine = EngineState::default();
-    engine
```

Comment:
Nothing interesting in this file. This fixes the benchmarks by adding the `meta` field to ClientFeatures that got missed recently and applies a `cargo fmt` to the file